### DISCUSSION TITLE
Why?

### DISCUSSION THREAD

**Discussion by Never-Developer** [cloudlink-omega/.github#3] [General]
![Ekran görüntüsü 2024-08-21 215401](https://github.com/user-attachments/assets/28817766-69e9-41e5-8666-b5144e474122)

**Comment by MikeDev101** [comment]
You need to verify your email address that you used to register an account with.

**Reply by Never-Developer** [reply]
Maybe servers down?

**Reply by MikeDev101** [reply]
I just tested on my end and the server is operating normally. Did you check for any messages in your browser's console log?

![Screenshot 2024-08-22 102120](https://github.com/user-attachments/assets/367d38dc-6e9f-454c-81c0-e57c9efbe512)

**Reply by Never-Developer** [reply]
It would be easier to talk about this on discord: never_dev

**Reply by MikeDev101** [reply]
Issue has been resolved via Discord. Marking as done.
